docs/examples/fmi_ls_manifest_example.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<MyRootElement 
+<MyRootElement
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:noNamespaceSchemaLocation="fmi_ls_manifest_example_schema.xsd"
   xmlns:fmi-ls="http://fmi-standard.org/fmi-ls-manifest"
   fmi-ls:fmi-ls-name="org.fmi-standard.demo-ls-name"
```

Comment:
Do we need the attribute prefix in addition to the namespace? Wouldn't "fmi-ls:name" be sufficient?

Reply:
It technically would, however it is probably still a good idea to have the more explicit names, given that in some circumstances the namespace is not readily apparent.